package.json

```diff
@@ -1,29 +1,27 @@
 {
   "name": "devicepilot",
   "description": "DevicePilot SDK for IoT Analytics",
-  "version": "1.2.5",
-  "main": "index.js",
+  "version": "2.0.0",
+  "main": "./dist/index.umd.js",
   "repository": "https://github.com/DevicePilot/devicepilot-js.git",
-  "homepage": "https://www.devicepilot.com",
-  "author": "thomasmichaelwallace <tom.wallace@devicepilot.com>",
-  "license": "MIT",
   "scripts": {
+    "build": "rollup -c",
     "test": "jest",
     "testing": "jest --watch"
   },
   "dependencies": {
-    "flat": "^4.1.0",
-    "request": "^2.88.0",
-    "request-promise-native": "^1.0.5"
+    "axios": "^0.19.0",
+    "flat": "^4.1.0"
   },
   "devDependencies": {
+    "babel-plugin-transform-es2015-modules-commonjs": "^6.26.2",
     "eslint": "^5.3.0",
     "eslint-config-airbnb-base": "^13.1.0",
     "eslint-plugin-import": "^2.14.0",
-    "jest": "^23.6.0"
-  },
-  "engines": {
-    "node": ">=7.6.0"
```

Comment:
Why did you remove the `engines` key?
I don't belive it does any harm when being imported for browser use, but does mean that we match the tested level of node.

Reply:
I had removed it because it was not applicable anymore before we added back URL polyfill

Reply:
I think it was - prior to 7.6 async/await is not supported by node.